controller/src/main/java/org/jboss/as/controller/extension/SubsystemInformation.java

```diff
@@ -56,4 +58,25 @@
      * @return the micro interface version, or {@code null} if the subsystem does not have a versioned interface
      */
     Integer getManagementInterfaceMicroVersion();
+
+
+    /**
+     * Gets full version of the subsystem
+     * @return full {@link ModelVersion}
+     */
+    default ModelVersion getManagementInterfaceVersion(){
+        return ModelVersion.create(getManagementInterfaceMajorVersion() != null ? getManagementInterfaceMajorVersion() : 0,
+                getManagementInterfaceMinorVersion() != null ? getManagementInterfaceMinorVersion() : 0,
+                getManagementInterfaceMicroVersion() != null ? getManagementInterfaceMicroVersion() : 0
+        );
+    }
+
+
+    default boolean isDeprecated(){
+        return false;
+    }
+
+    default boolean isHostCapable() {
+                return false;
+            }
```

Comment:
This is an unrelated change and no code is using this method. If we're going to make an unrelated change around this we should drop the getter/setter from SubsystemInformationImpl as no one uses the property.